db/helpers/seed_helper.rb

```diff
@@ -0,0 +1,29 @@
+class SeedHelper
+  def objectify(model)
+    return model.titleize.constantize unless model.include? "_"
```

Comment:
I don't think you'd need the `split&map`, you could use ActiveSupport::Inflector `.classify` or `.camelize`.